### PR TITLE
Migrate SDI Documentation from the Project's README on Github

### PR DESCRIPTION
**Pull request by toniedzwiedz**
Moving the documentation for Sling Dynamic Include to the Sling site, as suggested by @rombert in https://github.com/apache/sling-org-apache-sling-dynamic-include/pull/7

### CODE REVIEW

**Comment by rombert** [comment]
Thanks for the PR @toniedzwiedz ! I took a look at the rendered output ( via http://localhost:8820/documentation/bundles/dynamic-includes.html ) and here are some notes:

### 1. There is an ASF license notices after the TOC - I don't think you meant to include it

image

### 2. In the 'Apache & Dispatcher' section I think it would be good to indicate what the dispatcher is - it's not known to developers not using AEM .

image

Maybe something like : [AEM dispatcher module](https://helpx.adobe.com/experience-manager/dispatcher/using/dispatcher.html) ( commercial, requires AEM license ) ?

### 3. The link to the SDI request processing flow seems broken, can you please double-check?

image

Thanks!

**Comment by toniedzwiedz** [comment]
Hi @rombert, thanks for the feedback.

Ad 1) That was [already there on the page](https://sling.apache.org/documentation/bundles/dynamic-includes.html). I assumed it was there because the bundle was contributed to the ASF at some point.  If it's not necessary, I'm happy to remove it.

Ad 2) Fair enough, I'll rephrase it to provide a brief introduction with links.

Ad 3) The link I'm using is:

    SDI request processing flow

For some reason, given this input, the build generates a link with the site's public domain prepended. When I replace the host name with `localhost:8820`, leaving the rest of the path intact, the image loads as expected. That makes me believe the link will be correct once the site has been published.  Is the domain part configurable somewhere?

**Comment by rombert** [comment]
> 1) That was already there on the page. I assumed it was there because the bundle was contributed to the ASF at some point. If it's not necessary, I'm happy to remove it.

I assume this was a mistake, so please remove it.

> 2) Fair enough, I'll rephrase it to provide a brief introduction with links.

Sounds good to me.

> For some reason, given this input, the build generates a link with the site's public domain prepended. 

Looks like we have some problems with rendering images, so nothing for your to change. For the record, I filed [SLING-7840 - Image rendering broken on sling.apache.org](https://issues.apache.org/jira/browse/SLING-7840)

**Comment by toniedzwiedz** [comment]
@rombert I've introduced the changes you requested. Please review again.

**Comment by rombert** [comment]
Look great, thanks @toniedzwiedz ! Merged just now.

**Comment by toniedzwiedz** [comment]
Thanks @rombert, looking forward to seeing it deployed :)
